virttest/test_setup.py

```diff
@@ -2374,16 +2374,21 @@ def enable(self):
         if not self.log_file or not os.path.isdir(os.path.dirname(self.log_file)):
             self.log_file = utils_misc.get_path(self.test.debugdir,
                                                 "libvirtd.log")
+        if os.path.isfile(self.backupfile):
```

Comment:
After this change,it only keeps 'last' debug log when `enable()` is called.
What about make this optional?

Reply:
This is not the debug log, it is the libvirtd configuration file available in `/etc/libvirt/libvirtd.conf`
 - so clean any irrelevant file in same name if any before backing up
 - backup the current configuration file.
- change the configuration file and restart libvirtd.